### PR TITLE
Awesome books: refactor to use JavaScript classes

### PR DESCRIPTION
**Pull request by Hamzaoutdoors**
- Added Professional README file
- Added linters configuration (HTML, CSS and JS)
- Added book creation with object constructor
- Added fix for form submission
- Added event listener to add & remove books
- Added Class methods to remove book object from books array
- Added local storage functionality
- Added fix for local storage & fix linters errors

### CODE REVIEW

File: JS/main.js [hunk 1]
```diff
@@ -1,12 +1,5 @@
...
-class Book {
-  constructor(title, author) {
-    this.title = title;
-    this.author = author;
-  }
-}
-
 function getBooks() {
   myBooks = JSON.parse(localStorage.myBooks);
   const books = document.getElementById('books');
```
Comment (de-scholar) [inline JS/main.js]: - [x] All of the functionality of this project should be embedded inside classes, please move this function in your class

**Review by de-scholar — CHANGES_REQUESTED** [review]
Hi @Hamzaoutdoors and @petumazo , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: JS/main.js (hunk 1 above)
Comment (petudeveloper) [reply JS/main.js]: Issue solved, thanks for the feedback.

**Comment by petudeveloper** [comment]
@de-scholar Thanks for the feedback, we followed your suggestions and embedded all of the functionalities inside a class.

**Review by hemantso — APPROVED** [review]
Hi @Hamzaoutdoors and @petumazo, 

Well done on making the requested changes. 👍 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 

Alt Text

Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**